2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error updating stats
Exception: TypeError: object of type 'kivy.properties.ListProperty' has no len()
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 605, in update_display
    total_targets = len(self.app.app_state.tonights_targets)
TypeError: object of type 'kivy.properties.ListProperty' has no len()

2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error getting highlights content
Exception: TypeError: 'kivy.properties.ListProperty' object is not subscriptable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 891, in get_highlights_content
    top_targets = targets[:3]
TypeError: 'kivy.properties.ListProperty' object is not subscriptable

2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error updating stats
Exception: TypeError: object of type 'kivy.properties.ListProperty' has no len()
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 605, in update_display
    total_targets = len(self.app.app_state.tonights_targets)
TypeError: object of type 'kivy.properties.ListProperty' has no len()

2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error getting highlights content
Exception: TypeError: 'kivy.properties.ListProperty' object is not subscriptable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 891, in get_highlights_content
    top_targets = targets[:3]
TypeError: 'kivy.properties.ListProperty' object is not subscriptable

2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error updating stats
Exception: TypeError: object of type 'kivy.properties.ListProperty' has no len()
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 605, in update_display
    total_targets = len(self.app.app_state.tonights_targets)
TypeError: object of type 'kivy.properties.ListProperty' has no len()

2025-06-21 16:38:32 | ERROR    | AstroScope_errors | error:121 | Error getting highlights content
Exception: TypeError: 'kivy.properties.ListProperty' object is not subscriptable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 891, in get_highlights_content
    top_targets = targets[:3]
TypeError: 'kivy.properties.ListProperty' object is not subscriptable

2025-06-21 16:44:15 | ERROR    | AstroScope_errors | error:121 | Error updating stats
Exception: TypeError: 'kivy.properties.ListProperty' object is not iterable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 611, in update_display
    total_targets = len(list(targets_prop))
TypeError: 'kivy.properties.ListProperty' object is not iterable

2025-06-21 16:44:15 | ERROR    | AstroScope_errors | error:121 | Error getting highlights content
Exception: TypeError: 'kivy.properties.ListProperty' object is not iterable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 923, in get_highlights_content
    targets_list = list(targets)
TypeError: 'kivy.properties.ListProperty' object is not iterable

2025-06-21 16:44:15 | ERROR    | AstroScope_errors | error:121 | Error updating stats
Exception: TypeError: 'kivy.properties.ListProperty' object is not iterable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 611, in update_display
    total_targets = len(list(targets_prop))
TypeError: 'kivy.properties.ListProperty' object is not iterable

2025-06-21 16:44:15 | ERROR    | AstroScope_errors | error:121 | Error getting highlights content
Exception: TypeError: 'kivy.properties.ListProperty' object is not iterable
Traceback: Traceback (most recent call last):
  File "/Users/enricopapalini/Library/Mobile Documents/iCloud~com~omz-software~Pythonista3/Documents/astropy-app/mobile_app/screens/home_screen.py", line 923, in get_highlights_content
    targets_list = list(targets)
TypeError: 'kivy.properties.ListProperty' object is not iterable

